modules/list.nix

```diff
@@ -1,49 +1,50 @@
 [
-    ./runit
```

Comment:
I don't know what happened here but these changes should be unnecessary, I suspect "spaces -> tabs" or smth like it.